ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -181,6 +181,30 @@ router bgp {{ router_bgp.as }}
 {%          endfor %}
 {%       endif %}
 {%     endif %}
+{# address family rt-membership activation #}
+{%     if router_bgp.address_family_rtc is defined and router_bgp.address_family_rtc is not none %}
+   !
+   address-family rt-membership
+{%         if router_bgp.address_family_rtc.peer_groups is defined and router_bgp.address_family_rtc.peer_groups is not none %}
+{%            for peer_group in router_bgp.address_family_rtc.peer_groups | arista.avd.natural_sort %}
+{%               if router_bgp.address_family_rtc.peer_groups[peer_group].activate == true %}
+      neighbor {{ peer_group }} activate
+{%               elif router_bgp.address_family_rtc.peer_groups[peer_group].activate == false %}
+      no neighbor {{ peer_group }} activate
+{%               endif %}
+{%               if router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target is defined %}
+{%                   if router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.only is defined and router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.only == true %}
+      neighbor {{ peer_group }} default-route-target only
+{%                   else %}
+      neighbor {{ peer_group }} default-route-target
+{%                   endif %}
+{%               endif %}
+{%               if router_bgp.address_family_rtc.peer_groups[peer_group].default_route_target.encoding_origin_as_omit is defined %}
```

Comment:
Your Readme says `encording_origin_as_omit : < true | false >` but your test has `none`. The code also only looks if this knob is defined, so we could just change the readme to reflect that.

Reply:
Thanks, fixed!